1.8

```diff
@@ -0,0 +1,2 @@
+Requirement already satisfied: scipy in /home/drudis/anaconda3/envs/terra2/lib/python3.10/site-packages (1.7.3)
```

Comment:
This file I imagine was added in error

Reply:
Yes, indeed. I will remove it. However, note that the CI of Terra is loading scipy 1.7.3 but the newest version is 1.9.1. I have been going crazy trying to make the CI work because the tests were passing on my machine but not on the CI. 
The reason was that in 1.9.1 `scipy.sparse` has now a method `trace()`, and `expm_multiply` has an extra field `traceA` that allows you to pass the trace of the matrix(if you know it) to avoid having to compute it at each timestep.
In our case it doesn't matter since I have fixed it and benchmarked it and the performance difference is neglectable, but it might be worth it to update the requirements.txt file.

Reply:
Okay, good to know.